partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnEq.kt

```diff
@@ -16,8 +16,8 @@ import org.partiql.spi.value.Datum
  * > Equality never fails in the type-checking mode and never returns MISSING in the permissive mode. Instead, it can
  * compare values of any two types, according to the rules of the PartiQL type system. For example, 5 = 'a' is false.
  *
- * For the existing conformance tests, whenever an operand is NULL or MISSING, the output is NULL. This implementation
- * follows this.
+ * For the existing conformance tests, when an operand is NULL, the output is NULL. When an operand is MISSING,
```

Comment:
Hmm I believe the existing behavior is correct. Equality follows a bit different behavior than other functions/operators much like the logical operators. Perhaps there was a bad conformance test? Which test led to this change?

Reply:
Yes. Based on my read of spec section 7.1.1 Equality:
> Equality never fails in the type-checking mode and never returns `MISSING` in the permissive mode

`=` should never return `MISSING`.